quantlib/time/date.pyx

```diff
@@ -216,6 +216,18 @@ cdef class Period:
         else:
             return NotImplemented
 
+    def __itruediv__(self, value):
```

Comment:
huum, it sounds like it breaks on python3. That is really strange, the method is defined and there is no typo ...:

```
======================================================================
ERROR: test_inplace_division (quantlib.test.test_date.TestQuantLibPeriod)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/travis/build/enthought/pyql/quantlib/test/test_date.py", line 313, in test_inplace_division
    period /= 3
  File "quantlib/time/date.pyx", line 220, in quantlib.time.date.Period.__itruediv__ (quantlib/time/date.cpp:3149)
    self.__idiv__(value)
AttributeError: 'quantlib.time.date.Period' object has no attribute '__idiv__'
----------------------------------------------------------------------
```